crowdsourcing/serializers/project.py

```diff
@@ -49,17 +52,19 @@ def create(self, **kwargs):
             for item in template_items:
                 models.TemplateItem.objects.get_or_create(template=t[0], **item)
         if module.has_data_set:
```

Comment:
Did the `if` and the `else` part misplaced?

If `module.has_data_set` should create tasks but the code (to create such tasks) is in the `else` part.